src/onInit/defineVisitOrder.js

```diff
@@ -0,0 +1,44 @@
+import { set, ascending } from 'd3';
+
+export default function defineVisitOrder() {
+    this.config.time_cols.filter(time_col => time_col.type === 'ordinal').forEach(time_settings => {
+        let visits, visitOrder;
+
+        //Given an ordering variable sort a unique set of visits by the ordering variable.
+        if (time_settings.order_col && this.raw_data[0].hasOwnProperty(time_settings.order_col)) {
+            //Define a unique set of visits with visit order concatenated.
```

Comment:
NBD, but curious if there is a reason for the funky "|" concatenation here instead of just using an object with 2 properties?

Reply:
Because I don't think `d3.set()` works with objects?  Could be wrong about that.